.github/workflows/pr.yaml

```
@@ -7,21 +7,36 @@ on:
     branches: [main]
   workflow_dispatch:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
 jobs:
+  detect_changes:
```

actionlint (1.7.4)

job "detect_changes" needs job "wait_for_autofix" which does not exist in this workflow

(job-needs)